src/main/webapp/script.js

```diff
@@ -12,3 +12,25 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+function getCurrentUser() {
+  const params = new URLSearchParams(window.location.search);
+  uid = params.get('id');
+  console.log(uid);
+  return uid;
+}
+function setProfileLink() {
+  let uid = getCurrentUser();
+  var link = document.getElementById("profileLink");
+  link.setAttribute('href', 'profile.html?id=' + uid);
```

Comment:
Maybe: `link.setAttribute('href', 'profile.html?id=' + getCurrentUser);`

Reply:
i know my way is more code but i like the way its organized better!